articles/virtual-network/how-to-virtual-machine-mtu.md

```diff
@@ -208,7 +208,7 @@ Use the following steps to change the MTU size on a Linux virtual machine:
     An indication that there is a mismatch in settings between the source and destination displays as an error message in the output. In this case, the MTU isn't set on the source network interface.
 
     ```output
-    azureuser@vm-1:~/GetPathMTU$ ping 10.0.0.5 -c 10 -M do -s 3872
+    azureuser@vm-1:~/GetPathMTU$ ping 10.0.0.5 -c 10 -M do -s 3873
     PING 10.0.0.5 (10.0.0.5) 3872(3900) bytes of data.
```

Comment:
@mattreatMSFT

Can you also update the value in the examples so there isnt a discrepancy?

Thanks!